src/simulation/dynamics/facetSRPDynamicEffector/facetSRPDynamicEffector.cpp

```diff
@@ -0,0 +1,176 @@
+/*
+ ISC License
+
+ Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+ */
+
+#include "facetSRPDynamicEffector.h"
+#include <cmath>
+
+const double speedLight = 299792458.0; //  [m/s] Speed of light
+const double AstU = 149597870700.0; // [m] Astronomical unit
+const double solarRadFlux = 1368.0; // [W/m^2] Solar radiation flux at 1 AU
+
+/*! The constructor initializes the member variables to zero. */
+FacetSRPDynamicEffector::FacetSRPDynamicEffector()
+{
+    this->forceExternal_B.fill(0.0);
+    this->torqueExternalPntB_B.fill(0.0);
+    this->numFacets = 0;
+}
+
+/*! The destructor. */
+FacetSRPDynamicEffector::~FacetSRPDynamicEffector()
+{
+}
+
+/*! The reset member function. This method checks to ensure the input message is linked.
+ @return void
+ @param CurrentSimNanos [ns]  Time the method is called
+*/
+void FacetSRPDynamicEffector::Reset(uint64_t CurrentSimNanos)
+{
+    // Check if input message has not been included
+	if (!this->sunInMsg.isLinked()) {
```

Comment:
:small_red_triangle_down: Reduce the indenting here. Also need to go through this PR and check your indentation and formatting generally.

Reply:
Would you be able to point me to specific lines where the indentation is not correct? I'm not sure what you mean here. The code within the member functions of this module is indented once, is this not what you are looking for?

Reply:
I addressed the indenting, please let me know if there are any other indenting issues I missed.